src/benchq/algorithms/_qsp.py

```diff
@@ -89,12 +92,9 @@ def get_qsp_program(
         occ_state = np.zeros(pyliqtr_operator.problem_size)
         occ_state[0] = 1
 
-        steps = QSP.gen_qsp.get_phis(
-            pyliqtr_operator,
-            simtime=timestep_vec[1],
-            req_prec=required_precision,
-            steps_only=True,
-        )
+        # TODO: I think the way we calculate it is incorrect.
```

Comment:
What needs to be done to fix it? Is there a story on the board to investigate this?